docs/monitoring/metrics.md

```diff
@@ -335,6 +345,21 @@ Parameters:
 - `ttl` - time-to-live for transmitted UDP packets
 - `addressingMode` - UDP addressing mode to use (UNICAST/MULTICAST)
 
+Example configuration:
+
+{% highlight yaml %}
+
+metrics.reporters: gang
```

Comment:
If we don't expect users to read the full docs, it might actually be better if it's the full name after all.  😉 Otherwise they might think they have to do the abbreviation. Just saying 😄